include/ck/utility/inner_product.hpp

```diff
@@ -70,6 +70,12 @@ inner_product<float4_t, float4_t, float>(const float4_t& a, const float4_t& b, f
                   c);
 }
 
+template <>
+__device__ void inner_product<half_t, half_t, float>(const half_t& a, const half_t& b, float& c)
```

Comment:
where is this used?

Reply:
https://github.com/ROCmSoftwarePlatform/composable_kernel/blob/navi21_gemm/include/ck/tensor_operation/gpu/thread/threadwise_contraction_dlops.hpp#L210

Reply:
Do we really have cases where we use half_t (instead of half2_t, half4_t, etc) as datatype of ```a``` and ```b```?